packages/quantic/force-app/main/default/lwc/quanticCardContainer/quanticCardContainer.js

```diff
@@ -7,10 +7,37 @@ import { LightningElement, api } from 'lwc';
  * <c-quantic-card-container title="Card Example"></c-quantic-card-container>
  */
 export default class QuanticCardContainer extends LightningElement {
-    /**
-     * The title label to display in the card header.
-     * @api
-     * @type {string}
-     */
-    @api title;
-}
\ No newline at end of file
+  /**
+   * The title label to display in the card header.
+   * @api
+   * @type {string}
+   */
+  @api title;
+
+  /**
+   * Sets the focus on the card header.
+   * @api
+   * @type {VoidFunction}
+   */
+  @api setFocus() {
+    const focusTarget = this.template.querySelector('.card_focusable-header');
+    if (focusTarget) {
+      // @ts-ignore
```

Comment:
Why the ts-ignore?

Reply:
I'd wager the object returned by lwc querySelector doesn't have `focus` defined.

Reply:
exactly 👆

Reply:
Yea but can't we export the type and import here?

Reply:
We can do something like this to avoid the ts-ignore:
```
/** @type {Element & {focus: function}} */
const focusTarget = this.template.querySelector('.card_focusable-header');
```